src/widget.cpp

```diff
@@ -347,6 +350,7 @@ void widget::load( const JsonObject &jo, const std::string & )
     optional( jo, was_loaded, "height", _height_max, 1 );
     optional( jo, was_loaded, "symbols", _symbols, "-" );
     optional( jo, was_loaded, "fill", _fill, "bucket" );
+    optional( jo, was_loaded, "separator", _separator, default_separator );
```

Comment:
Is there a reason to not just make it mandatory, so we don't have a default you should never use?

Reply:
Is there a way to make it mandatory for just the `sidebar` style of widget? If so, we can do that.

Reply:
I have made this field mandatory if the widget style is `sidebar`. This should prevent the default from appearing in custom sidebars.